docs/UserGuide.md

```diff
@@ -78,48 +86,49 @@ Shows a message explaining how to access the help page.
 Format: `help`
 
 
-### Adding a client: `add`
+### Adding a client : `add`
 
 Adds a client to the HustleBook.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [a/ADDRESS] [d/DATA]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [f/FLAG] [i/INFO] [d/DATE] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A client's data can be added in the future
+A client's data can be added in the future through edit command
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 d/Salary-3400`
-
-
-
-### Adding additional information: `addinfo`
-
-Adds additional information that is relevant to a client
-
-Format: `Format: addinfo n/NAME [INFO] [VALUE]`
-
+* `d/DATE` will be set to today's date by default if not specified.
+* `i/INFO` will be set to `No further info` by default if not specified.
+* `t/TAG` will be empty by default if not specified.
+* `f/FLAG` will be set to `false` by default if not specified.
 
 Examples:
-* `addinfo n/John Doe Preference CryptoCurrencies`
-* `addinfo n/Betsy Risk High`
+* `add n/John Doe p/98765432 e/johnd@example.com a/Blk 775 Pasir Ris Street 71 S510775`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/NUS School of Computing, COM1 p/1234567 i/Salary of $3400 f/true`
 
 ### Listing all persons : `list`
 
 Shows a list of all clients in the HustleBook.
 
 Format: `list`
 
+### Flagging a person : `flag`
+
+Flag a person in the HustleBook to mark them as important. Can also be used to unflag the person.
+
+Format: `flag INDEX f/FLAG`
+
+* `FLAG` input should either be true or false.
+* `True` or `False` is not case-sensitive
+
 ### Editing a person : `edit`
 
 Edits an existing person in the HustleBook.
 
-* Using **Index**:
+* Using **Name**:
 
-  Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+  Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/INFO] [d/DATE] [t/TAG]…​`
 
-  * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+  * Edits the person named `NAME`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
It would be better if the second statement can be deleted as there is no more index that can be used for this command.

Reply:
Okay got it 😄